doc/classes/AudioEffectDistortion.xml

```diff
@@ -82,26 +86,35 @@
 	</methods>
 	<members>
 		<member name="drive" type="float" setter="set_drive" getter="get_drive">
+			Power of distortion. Value can range from 0 to 1. Default value: [code]0[/code].
 		</member>
 		<member name="keep_hf_hz" type="float" setter="set_keep_hf_hz" getter="get_keep_hf_hz">
+			High-pass filter, frequencies higher than this value will not be affected by the distortion. Value can range from 1 to 20000. Default value: [code]16000[/code].
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="AudioEffectDistortion.Mode">
+			Type of distortion. Default value: [code]Clip[/code].
 		</member>
 		<member name="post_gain" type="float" setter="set_post_gain" getter="get_post_gain">
+			Increases or decreases the volume after the effect. Value can range from -80 to 24. Default value: [code]0[/code].
 		</member>
 		<member name="pre_gain" type="float" setter="set_pre_gain" getter="get_pre_gain">
+			Increases or decreases the volume before the effect. Value can range from -60to 60. Default value: [code]0[/code].
 		</member>
 	</members>
 	<constants>
 		<constant name="MODE_CLIP" value="0">
+			Digital distortion effect wich cuts off peaks at the top and bottom of the waveform.
 		</constant>
 		<constant name="MODE_ATAN" value="1">
 		</constant>
 		<constant name="MODE_LOFI" value="2">
+			Low-resolution digital distortion effect. You can use it to emulate the sound of early digital audio devices.
 		</constant>
 		<constant name="MODE_OVERDRIVE" value="3">
+			Emulates the warm distortion produced by a field effect transistor, which is commonly used in solid-state musical instrument amplifiers.
 		</constant>
 		<constant name="MODE_WAVESHAPE" value="4">
+			Waveshapers distortion are used mainly by electronic musicians to achieve an extra-abrasive sound.
```

Comment:
Tense is off here. "Waveshaper distortions *are* used" or "Waveshaper distortion *is* used"